.github/workflows/dev-cd.yml

```
@@ -0,0 +1,107 @@
+name: Deploy to Development Server
+
+on:
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
+    inputs:
+      image_tag:
+        description: "Docker 이미지 태그"
+        required: false
+        default: "dev-${{ github.sha }}"
```

actionlint (1.7.7)

context "github" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)